bazel/envoy_test.bzl

```diff
@@ -96,7 +96,7 @@ def envoy_cc_fuzz_test(
         srcs = tar_src,
         testonly = 1,
     )
-    fuzz_copts = ["-DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION"]
+    fuzz_copts = []
```

Comment:
we can just remove this now that it's empty

Reply:
sure, was just keeping it in case someone wanted to add something in the future but I guess it's better to remove it for now